x-pack/plugins/security_solution/cypress/integration/timelines/open_timeline.spec.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { timeline } from '../../objects/timeline';
+
+import { TIMELINE_DESCRIPTION, TIMELINE_TITLE, OPEN_TIMELINE_MODAL } from '../../screens/timeline';
+import {
+  TIMELINES_DESCRIPTION,
+  TIMELINES_PINNED_EVENT_COUNT,
+  TIMELINES_NOTES_COUNT,
+  TIMELINES_FAVORITE,
+} from '../../screens/timelines';
+import { cleanKibana } from '../../tasks/common';
+
+import { loginAndWaitForPage } from '../../tasks/login';
+import { openTimelineUsingToggle } from '../../tasks/security_main';
+import {
+  addFilter,
+  addNameAndDescriptionToTimeline,
+  addNotesToTimeline,
+  closeOpenTimelineModal,
+  closeTimeline,
+  markAsFavorite,
+  openTimelineFromSettings,
+  pinFirstEvent,
+  populateTimeline,
+  waitForTimelineChanges,
+} from '../../tasks/timeline';
+
+import { OVERVIEW_URL } from '../../urls/navigation';
+
+describe('Open timeline', () => {
+  before(() => {
+    cleanKibana();
+    loginAndWaitForPage(OVERVIEW_URL);
```

Comment:
If the focus of this spec is just to see that things are properly displayed, we should create the timeline with the API in order to speed up the execution of the test.